index.js

```diff
@@ -131,17 +136,30 @@ AudioManager.prototype.setVolume = function (channelId, volume, muted) {
 		// a sound was playing, channel becomes muted
 		channel.loopSound.stop();
 		// TODO: unload sound ?
-	} else if (channel.loopSound) {
-		// a sound is loaded in channel, updating volume & playback
+	} else if (channel.loopSound && channel.loopSound.id === channel.loopId) {
+		// correct sound is loaded in channel, updating volume & playback
 		channel.loopSound.setVolume(Math.max(0, Math.min(1, volume * channel.loopVol)));
 		if (wasChannelMuted) { channel.loopSound.play(); }
 	} else if (!channel.muted) {
-		// no sounds are loaded in channel, channel is unmutted
+		// sound is not loaded in channel, channel has been unmutted
 		this.playLoopSound(channelId, channel.loopId, channel.loopVol);
 	}
 };
 
 //▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
+/** Mute / unmute all channels
+ *
+ * @param {boolean} [muted] - Should all channels be muted. If not specified, function will behave as toggle
+ */
+AudioManager.prototype.muteAll = function (muted) {
```

Comment:
I was concern that `mute` could be seen as a function to mute only one particular channel, hence the `allMute` name. 
Now, i'm thinking about factorise the channels as a separate class to solve this problem.

I don't have better name so I think I'll go for `setMute(bool)`